server-spi/src/main/java/org/keycloak/locale/LocaleSelectorProvider.java

```diff
@@ -24,7 +24,6 @@
 
 public interface LocaleSelectorProvider extends Provider {
 
-    String LOCALE_COOKIE = "KEYCLOAK_LOCALE";
```

Comment:
Not sure we can delete constant from the interface inside `server-spi`? Especially when considering that `LocaleSelectorSpi` is not internal SPI. Should it have deprecation period or something?

Reply:
Added it back with a @Deprecated annotation